libs/shared/src/const.ts

```diff
@@ -26,6 +27,24 @@ export const ChainNames: Record<SupportedChainId, string> = {
   [SupportedChainId.SEPOLIA]: 'Sepolia',
 };
 
+// TODO: Get from SDK
+export const EXPLORER_NETWORK_NAMES: Record<SupportedChainId, string> = {
+  [SupportedChainId.MAINNET]: 'mainnet',
+  [SupportedChainId.ARBITRUM_ONE]: 'arb1',
+  [SupportedChainId.GNOSIS_CHAIN]: 'gc',
```

Comment:
OCD nitpick:

```suggestion
  [SupportedChainId.GNOSIS_CHAIN]: 'gc',
  [SupportedChainId.ARBITRUM_ONE]: 'arb1',
```

Just to have is the same order as the other 2 maps